packages/palette-tokens/src/helpers.ts

```diff
@@ -1,4 +1,5 @@
-import tokens, { Color, ColorValue, SpacingUnit } from "./index";
+import { THEME_V3 } from ".";
```

Comment:
This will break

Reply:
why?

Reply:
wow i had to fix a bunch of those. how.. how are they breaking even. feels like a bad configuration. I've never seen this break anywhere else.